source/install/install-kubernetes.rst

```diff
@@ -1,110 +1,100 @@
+* **Object Storage:**  An S3-compatible object storage service, such as Amazon S3, or similar. This is ideal for storing Mattermost file uploads and attachments. While other storage solutions are possible, S3-compatible storage offers the best scalability, reliability, and cost-effectiveness in most cloud and on-premise Kubernetes deployments.
+* **Managed Database:** A managed database service, such as Amazon RDS for PostgreSQL, Azure Database for PostgreSQL, or Google Cloud SQL for PostgreSQL. These services provide automated backups, high availability, and simplified database administration.
+* **Ingress Controller:** An Ingress controller, such as NGINX, to route external traffic to your Mattermost instance. This is essential for enabling external access to your Mattermost deployment. 
 
-.. tip::
-
-    - If you’re unsure about which environment you want to use for your Kubernetes cluster, we suggest using a managed service such as as `Amazon EKS <https://aws.amazon.com/eks/>`_, `Azure Kubernetes Service <https://azure.microsoft.com/en-ca/products/kubernetes-service/>`_, `Google Kubernetes Engine <https://cloud.google.com/kubernetes-engine/>`_, or `DigitalOcean Kubernetes <https://www.digitalocean.com/products/kubernetes/>`_.
-    - Looking for a quick way to evaluate Mattermost or to try out the latest Mattermost deployment? Learn how to `create your own Mattermost instance on Kubernetes using Minicube <https://medium.com/@Erez.Tamam/create-your-own-mattermost-instance-on-kubernetes-in-10-minutes-d13f576ed794>`_.
-    - See the :doc:`configuration settings </configure/configuration-settings>` documentation to learn more about customizing your production Mattermost deployment.
-    - Encountering issues with your deployment? See the :ref:`deployment troubleshooting <install/troubleshooting:deployment troubleshooting>` documentation for details.
+While this guide focuses on using external, managed services for your database and file storage, the Mattermost Operator *does* offer the flexibility to use other solutions. For example, you could choose to deploy a PostgreSQL database within your Kubernetes cluster using the CloudNative PG operator, or use a self-hosted MinIO instance for object storage. 
 
-Install Mattermost on Kubernetes
---------------------------------
+However, managing these components yourself adds complexity and is generally not recommended for production deployments unless you have specific operational requirements. This guide will primarily focus on the recommended approach of using S3-compatible storage and a managed database service. If you choose to use self-managed components, you will need to adapt the instructions accordingly.
```

Comment:
I like how we are more opinionated of how to install and run Mattermost via the operator, but I think it would be good to walk back some of this wording a bit. In particular `generally not recommended for production deployments` should probably be clarified so our customers that want to own all of their data know they are using a supported setup.

Reply:
Updated